http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorRoutes.scala

```diff
@@ -55,5 +55,14 @@ class CollectorRoutes[F[_]: Sync](collectorService: Service[F]) extends Http4sDs
       )
   }
 
-  val value: HttpApp[F] = (healthRoutes <+> corsRoute <+> cookieRoutes).orNotFound
+  def rejectRedirect = HttpRoutes.of[F] {
+    case _ -> Root / "r" / _ =>
+      NotFound("redirects disabled")
```

Comment:
>  Is there no way to manage the route directly like we do with other paths, without using a function ?
E.g.     case _ -> Root / "r" / _ if (!enableDefaultRedirect) =>

I think current way better because you check `enableDefaultRedirect` just one time. Otherwise, you have to check that flag with all redirect requests.

>  In case of Root / "r" / _ with redirecte enabled this is handled by Root / vendor / version right ?

Yes

> What if the URI is just Root / "r" ?

It would return 404 because it isn't handled.